_inc/client/plans/index.jsx

```diff
@@ -3,29 +3,79 @@
  */
 import React from 'react';
 import { connect } from 'react-redux';
+import { get } from 'lodash';
 
 /**
  * Internal dependencies
  */
-import { getSiteConnectionStatus } from 'state/connection';
 import PlanGrid from './plan-grid';
+import { SingleProductBackup } from './single-product-backup';
 import QuerySite from 'components/data/query-site';
+import { getAvailablePlans, getSitePlan } from 'state/site';
 
 export class Plans extends React.Component {
 	render() {
+		const { sitePlans } = this.props;
+
 		return (
-			<div>
+			<>
 				<QuerySite />
-				<div>
-					<PlanGrid />
-				</div>
-			</div>
+				{ 'jetpack_free' === get( this.props.sitePlan, 'product_slug', 'jetpack_free' ) &&
```

Comment:
Only until the other states in the Figma are implemented. In the interest of trying to keep this in a shippable state I added this check so that premium/pro plans will be unaffected. However they will get their own states in a followup PR.

Reply:
I'm not sure what you mean, could you please elaborate?

Reply:
Sorry, I think I misunderstood your question. It defaults to `jetpack_free` because the object is empty for new sites on a free plan. This was how other places in the code checked for the free plan too, so I assumed that some sites will actually return a free plan instead of an empty object.

Reply:
I changed this to have no default because I found a case where the card would "flash" if the page loaded at the right speed, and removing the default prevents this from happening.

Reply:
I guess you saw why I was asking this 😉 Thanks for addressing.